index.html

```diff
@@ -55,6 +74,276 @@
 
     -->
 
-    <h1>Assignment 01 - To Be Completed</h1>
-  </body>
+
+<header>
+  <h2>Apple Inc.</h2>
+
+  <img src="./images/apple.png" alt="apple logo" >
+
+
+<nav>
+  <h2>Apple Menu</h2>
+  <ul>
+      <li><a href="#">Home</a></li>
+      <li><a href="#">About us</a> </li>
```

Comment:
I'm going to challenge you on this heading structure - you already have an `h2` in the `header` so your second h2 is a bit wonky here.